tests/flytekit/unit/use_scenarios/unit_testing/test_type_hints.py

```diff
@@ -172,6 +172,28 @@ def my_wf(a: int, b: str) -> (int, str):
     }
 
 
+def test_wf1_with_list_of_inputs():
```

Comment:
we should also probably add a test that says 
"workflow.serialize()"
"task.serialize()"